doc/api/fs.md

```diff
@@ -725,19 +725,55 @@ added: v0.8.6
 -->
 
 * `fd` {Integer}
-* `len` {Integer}
+* `len` {Integer} default = `0`
 * `callback` {Function}
 
 Asynchronous ftruncate(2). No arguments other than a possible exception are
 given to the completion callback.
 
+If the file referred by the file descriptor has more number of bytes than the
+`len` then, only the first `len` bytes will be retained in the file.
+
+For example, the following program retains only the first four bytes of the file
+
+```
+> fs.readFileSync('temp.txt', 'utf8')
+'Node.js'
+> const fd = fs.openSync('temp.txt', 'r+')
+undefined
+> fs.ftruncate(fd, 4, (err) => {
+    assert.ok(!err);
+    console.log(fs.readFileSync('temp.txt', 'utf8'));
+  });
+undefined
+> Node
+```
```

Comment:
```` ```js ````?

I also think it might be nicer to have this as a script that could be run directly rather than a `REPL` excerpt, but that’s obviously a question of personal taste.